lib/node_modules/@stdlib/stats/base/dstdev/README.md

```diff
@@ -224,6 +208,125 @@ console.log( v );
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/stats/base/dstdev.h"
+```
+
+#### stdlib_strided_dstdev( N, correction, \*X, strideX )
+
+Computes the [standard deviation][standard-deviation] of a double-precision floating-point strided array.
+
+```c
+const double x[] = { 1.0, -2.0, 2.0 };
+
+double v = stdlib_strided_dstdev( 3, 1.0, x, 1 );
+// returns 2.5
```

Comment:
This isn't correct.

@0PrashantYadav0 Please go through this PR and more closely determine whether further things need updated.